src/main/java/com/synopsys/integration/detect/lifecycle/run/step/IntelligentModeStepRunner.java

```diff
@@ -183,7 +201,7 @@ private void checkPolicy(ProjectVersionView projectVersionView, BlackDuckRunData
     public void waitForCodeLocations(CodeLocationWaitData codeLocationWaitData, NameVersion projectNameVersion, BlackDuckRunData blackDuckRunData)
         throws OperationException {
         logger.info("Checking to see if Detect should wait for bom tool calculations to finish.");
-        if (operationRunner.createBlackDuckPostOptions().shouldWaitForResults() && codeLocationWaitData.getExpectedNotificationCount() > 0) {
+        if (codeLocationWaitData.getExpectedNotificationCount() > 0) {
```

Comment:
Looks like this method is no longer used?

Reply:
yes I believe that is correct. With the core work reviewed I'll start on the removal approach which I had avoided initially as I believe it will be quiet noisy in terms of the diffs.

Reply:
oh good idea!